src/nvim/screen.c

```diff
@@ -5278,6 +5129,43 @@ win_redr_custom (
   entered = FALSE;
 }
 
+static void sc_from_ascii(char_u *p, const char c)
```

Comment:
`sc` as in `screenchar` ?

Reply:
Maybe `schar` is better in analogy with `schar_T`? Or, we could rename the type to something with `cell` in it.